crates/ruma-events/tests/it/event_enums.rs

```diff
@@ -46,3 +53,53 @@ fn deserialize_message_event() {
     assert_eq!(content.call_id, "foofoo");
     assert_eq!(content.version, VoipVersionId::V0);
 }
+
+#[test]
+fn text_msgtype_plain_text_deserialization_as_any() {
+    let serialized = json!({
+        "body": "Hello world!",
+        "msgtype": "m.text"
+    });
+
+    let raw_event: Raw<AnyMessageLikeEventContent> =
+        Raw::from_json_string(serialized.to_string()).unwrap();
+
+    let event = raw_event.deserialize_with_type("m.room.message".into()).unwrap();
+
+    assert_matches!(event, AnyMessageLikeEventContent::RoomMessage(content));
+    assert_eq!(content.body(), "Hello world!");
+}
+
+#[test]
+fn text_msgtype_secret_storage_key_text_deserialization_as_any() {
```

Comment:
Nit: the test name looks wrong, this is not a msgtype, and there is no "text" variant.

Reply:
oh wow that copy-pasto fail. Thanks.